typings/pencil-code-embed-defs.d.ts

```diff
@@ -0,0 +1,88 @@
+// Code - https://pencilcode.net/lib/pencilcodeembed.js
+// Type definitions are only added for PencilCodeEmbed class
+// since it is the only class used in our codebase.
+
+interface SetupCode {
+  code: String,
+  type: String
+}
+
+class PencilCodeEmbed {
+  div: Object;
+  updatedCode: String;
+  callbacks: Object;
+  requests: Object;
+
+  // Hook up noop event handlers
```

Comment:
Add fullstop in end of all comments.

Here & below.

Reply:
Done!